plugins/cache_promote/configs.cc

```diff
@@ -48,7 +48,7 @@ bool
 PromotionConfig::factory(int argc, char *argv[])
 {
   while (true) {
-    int opt = getopt_long(argc, (char *const *)argv, "psbhe", longopt, nullptr);
```

Comment:
The short options value is needed if you want to allow sort options.

However, the short options are not documented:
https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/cache_promote.en.html?highlight=promotion